### ISSUE TITLE
After suspend, no sound until Pulseeffects is killed

### ISSUE THREAD

**Issue by Tooniis** [wwmm/easyeffects#911]
I can't open it either before killing it; running `pulseeffects` hangs and does nothing.

```
$ G_MESSAGES_DEBUG=pulseeffects pulseeffects --gapplication-service
(process:245761): pulseeffects-DEBUG: 12:22:19.359: main: locale directory: /usr/share/locale
(pulseeffects:245761): pulseeffects-DEBUG: 12:22:19.438: presets_manager: system input presets directory: "/etc/PulseEffects/input"; 
(pulseeffects:245761): pulseeffects-DEBUG: 12:22:19.438: presets_manager: system input presets directory: "/etc/xdg/PulseEffects/input"; 
(pulseeffects:245761): pulseeffects-DEBUG: 12:22:19.438: presets_manager: system output presets directory: "/etc/PulseEffects/output"; 
(pulseeffects:245761): pulseeffects-DEBUG: 12:22:19.438: presets_manager: system output presets directory: "/etc/xdg/PulseEffects/output"; 
(pulseeffects:245761): pulseeffects-DEBUG: 12:22:19.440: presets_manager: user presets directory already exists: /home/$USER/.config/PulseEffects
(pulseeffects:245761): pulseeffects-DEBUG: 12:22:19.441: presets_manager: user presets directory already exists: /home/$USER/.config/PulseEffects/input
(pulseeffects:245761): pulseeffects-DEBUG: 12:22:19.441: presets_manager: user presets directory already exists: /home/$USER/.config/PulseEffects/output
(pulseeffects:245761): pulseeffects-DEBUG: 12:22:19.441: presets_manager: user presets directory already exists: /home/$USER/.config/PulseEffects/autoload
Failed to register: Unable to acquire bus name 'com.github.wwmm.pulseeffects'
(pulseeffects:245761): pulseeffects-DEBUG: 12:22:19.442: application:  destroyed
(pulseeffects:245761): pulseeffects-DEBUG: 12:22:19.442: presets_manager: destroyed
```

Arch Linux, `5.11.2-arch1-1`
Pipewire 0.33.2
Pulseffects 5.0.0, installed with pacman

**Comment by wwmm** [comment]
So even after moving to PipeWire we still have problems in some computers when they are suspended...

>Failed to register: Unable to acquire bus name 'com.github.wwmm.pulseeffects'

Was this line shown when you tried to open a window when the command above was running?

**Comment by Tooniis** [comment]
> Was this line shown when you tried to open a window when the command above was running?

Pulseeffects was running (or rather frozen) in the background when I ran `G_MESSAGES_DEBUG=pulseeffects pulseeffects --gapplication-service` and got that. I checked again and it happens even when it's working properly so I think that isn't the right way to get logs.

**Comment by wwmm** [comment]
>I think that isn't the right way to get logs

You have to run `pulseeffects -q` first. This will kill the current instance. This is needed to get logs.

**Comment by wwmm** [comment]
I think things are so different now in EasyEffects that it will be better to open a new issue for this if this problems persists. Just like in the older release suspending on my computer causes nothing unusual.
